server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/HotRodRollingUpgradesIT.java

```diff
@@ -33,30 +33,7 @@
  */
 @RunWith(Arquillian.class)
 @Category({RollingUpgrades.class})
```

Comment:
If they run by default, they will only test a rolling upgrade from "current" to "current" version. You would need to supply a path to an older server version that will be automatically unzipped. But that complicates running the suite in CI.